src/engines/html5/html5.js

```diff
@@ -253,17 +253,7 @@ export default class Html5 extends FakeEventTarget implements IEngine {
    */
   attachMediaSource(): void {
     if (this._mediaSourceAdapter) {
-      //added to mask problematic behavior in shaka - init fire loadstart event, only for last init to avoid spinner
       this._mediaSourceAdapter.attachMediaSource(this._playbackEndedOnDetach);
-      if (this._playbackEndedOnDetach) {
```

Comment:
@Yuvalke so what is the solution here if you remove this?

Reply:
Shaka init on load, so i didn't init on attaching cause when load call it'll be init again.
On the end load doesn't happen

Reply:
Good news, so we can wrap this with the other PRs or anything else is left?

Reply:
I have a small fix which will remove the parameters, there are not needed anymore.
Every adapter can handle every by himeself

Reply:
I have a small fix which will remove the parameters, there are not needed anymore.
Every adapter can handle every by himeself